packages/core/src/app-middlewares/after/large-response-cacher.js

```diff
@@ -6,6 +6,9 @@ const responseStasher = require('../../tools/create-response-stasher');
 
 const largeResponseCachePointer = async (output) => {
   const response = cleaner.maskOutput(output);
+  if (!response.results) {
+    return output;
+  }
```

Comment:
Semantically, any concern with `response.results` being JavaScript "falsy"? Like an empty string.

Reply:
Yeah! The only case we want to move forward is if there's a big payload, so if it's falsy, we'll just carry on and have the next middleware deal with the `output`